presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/StaticSelector.java

```diff
@@ -75,7 +78,7 @@ public StaticSelector(
     }
 
     @Override
-    public Optional<SelectionContext<VariableMap>> match(SelectionCriteria criteria)
+    public Optional<SelectionContext<VariableMap>> match(SelectionCriteria criteria, ConcurrentMap<ResourceGroupId, ResourceGroup> groups)
```

Comment:
From the method signature perspective this doesn't make much sense :) 

We should change the return type for this method to be a new class - maybe `SelectorResult` and then the ResourceGroupConfigurationManager - can read the data from `SelectorResult`, get the limits and return a `SelectionContext`